ui/src/utils/constants.js

```diff
@@ -7,6 +7,7 @@ export const QUESTIONANALYSIS = 'QUESTIONANALYSIS';
 export const XVSY = 'XVSY';
 export const SESSIONS = 'SESSIONS';
 export const STATEMENTS = 'STATEMENTS';
+export const JOURNEY_PROGRESS = 'JOURNEY_PROGRESS';
```

Comment:
This probably only belongs in Enterprise, but I don't mind this constant being in this PR.